frontend/src/App.tsx

```diff
@@ -46,6 +47,12 @@ if (import.meta.env.PROD) {
 	});
 }
 
+const customNavigate = (path: string) => {
+	window.history.pushState({}, "", path);
+
+	window.dispatchEvent(new Event("popstate"));
+};
+
```

Comment:
How about add comment `TODO: Change it to navigate`?
I think we should change it in the future.

Reply:
I agree with that. However, if you want to use navigate within app.tsx, useNavigate() must be available only in the context of the <Router> component.
So, I think it would be good to resolve this issue in the future.